### ISSUE TITLE
Fresh install on Windows7 gets: "File association not found for extension .py"

### ISSUE THREAD

**Issue by peternann** [aws/aws-cli#3548]
Just installed AWS CLI on Windows 7, using python/pip method described here:
https://docs.aws.amazon.com/cli/latest/userguide/awscli-install-windows.html#awscli-install-windows-pip

Installed Python 3.6.6, and then as per instructions.

Get this output running aws:
```
C:\Users\...>aws --version
File association not found for extension .py
aws-cli/1.16.9 Python/3.6.6 Windows/7 botocore/1.11.9
```

Why the 'File Association' line?

Found that this fixes the issue:
```
assoc .py=py_auto_file
ftype py_auto_file="%USERPROFILE%\AppData\Local\Programs\Python\Python36\python.exe" "%1" %*
```

But why is that needed after a clean install via the documentation?

**Comment by justnance** [comment]
@peternann - Thanks for reaching out and for your patience.  We don't know why this happens and appears to be an intermittent problem.  There are some other issues that discuss this topic in more detail:  [243](https://github.com/PmagPy/PmagPy/issues/243) and [83](https://github.com/awslabs/aws-shell/issues/83).

**Comment by justnance** [comment]
@peternann - After digging a little deeper, I found this is related to #3518.  I'm going to leave this issue closed so the solution can be tracked under #3518.
